db/migrate/20230623080252_create_recipes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string :name, null: false
+      t.text :description
+      t.string :thumbnail
+      t.integer :serving_size
+      t.boolean :is_draft
```

Comment:
booleanに対してはデフォルト値とnull制御を入れておくと良いです！
```suggestion
      t.boolean :is_draft, default: false, null: false
```

Reply:
修正しました！